tests/models/test_gpu.py

```diff
@@ -93,6 +98,35 @@ def test_multi_gpu_model_dp(tmpdir):
     memory.get_memory_profile('min_max')
 
 
+@pytest.mark.parametrize('cli_args', [
+    pytest.param('--max_epochs 1 --gpus 2 --distributed_backend ddp'),
+])
+@pytest.mark.parametrize('variation', train_test_variations.get_variations())
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+def test_multi_gpu_model_ddp(tmpdir, cli_args, variation):
+    file = Path(train_test_variations.__file__).absolute()
+    cli_args = cli_args.split(' ') if cli_args else []
+    cli_args += ['--default_root_dir', str(tmpdir)]
+    command = [sys.executable, file, '--variation', variation] + cli_args
+    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
@Borda I am launching a script here, and I am using `sys.executable` to get the correct python interpreter, but in CI I get the error ModuleNotFoundError, pytorch_lightning not found. Any idea how this works in CI?

Reply:
can you print what is the executable?

Reply:
@Borda I'm already printing it, it's a hard to find in the output, it shows `/opt/conda/bin/python`

Reply:
@Borda I found what was the problem. I had to add pytorch_lightning manually to PYTHONPATH because pip freeze didn't show the package as installed. Now the test is running and freezing as expected.